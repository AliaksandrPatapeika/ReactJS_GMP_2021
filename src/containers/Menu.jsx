import '../less/containers/Menu.less';

import React from 'react';

import ResultsSort from '../components/ResultsSort';
import ResultsFilter from './ResultsFilter';

const Menu = () => (
  <div className="menuContainer">
    <ResultsFilter />
    <ResultsSort />
  </div>
);

export default Menu;
