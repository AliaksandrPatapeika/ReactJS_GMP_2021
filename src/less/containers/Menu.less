@import "../variables";

.menuContainer {
	display: flex;
	justify-content: space-between;
	border-bottom: 3px solid @dark-gray;
	height: 66px;
	padding-top: 16px;
}