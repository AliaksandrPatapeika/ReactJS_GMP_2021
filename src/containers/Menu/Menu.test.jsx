import React from 'react';
import renderer from 'react-test-renderer';

import Menu from './Menu';

describe('Menu', () => {
  test('renders correctly', () => {
    const menu = renderer
      .create(<Menu />)
      .toJSON();

    expect(menu).toMatchSnapshot();
  });
});
