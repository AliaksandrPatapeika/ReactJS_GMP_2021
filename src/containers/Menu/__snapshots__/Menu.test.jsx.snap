// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<div
  className="menuContainer"
>
  <nav>
    <ul
      className="resultsFilterContainer"
    >
      <li
        className="filterItem active"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        ALL
      </li>
      <li
        className="filterItem"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        DOCUMENTARY
      </li>
      <li
        className="filterItem"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        COMEDY
      </li>
      <li
        className="filterItem"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        HORROR
      </li>
      <li
        className="filterItem"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="menuitem"
        tabIndex="0"
      >
        CRIME
      </li>
    </ul>
  </nav>
  <div
    className="resultsSortContainer"
  >
    <div
      className="sortBy"
    >
      SORT BY
    </div>
    <div
      className="releaseDate"
    >
      RELEASE DATE
    </div>
    <button
      className="arrow"
      onClick={[Function]}
      type="button"
    >
      
      <img
        alt="Sort Button"
        src="test-file-stub"
      />
    </button>
  </div>
</div>
`;
